string type_id
uuid_msgs/UniqueID uuid
float64 confidence
geometry_msgs/PoseStamped pose_stamped
